src/azure-cli/azure/cli/command_modules/configure/tests/latest/test_configure.py

```diff
@@ -56,5 +57,13 @@ def test_configure_global_defaults(self):
         self.assertTrue(actual == expected)
 
 
+class LocalContextTest(ScenarioTest):
```

Comment:
Maybe should belong to `src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_local_context.py`?